composer.json

```diff
@@ -16,7 +16,10 @@
   },
   "require": {
     "php": ">=7.1",
-    "phpmd/phpmd": "^2.4"
+    "phpmd/phpmd": "^2.4",
+    "symfony/finder": "*",
+    "zendframework/zend-config": "*",
+    "roave/better-reflection": "*"
```

Comment:
Are those also cleared by legal? Are we using them somewhere else already?

Reply:
We use those packages in spryker/spryker and spryker/spryk